mappings/net/minecraft/world/biome/BiomeEffects.mapping

```diff
@@ -3,15 +3,48 @@ CLASS net/minecraft/class_4763 net/minecraft/world/biome/BiomeEffects
 	FIELD field_22068 waterColor I
 	FIELD field_22069 waterFogColor I
 	FIELD field_22070 particleConfig Ljava/util/Optional;
+	FIELD field_22491 loopSound Ljava/util/Optional;
+	FIELD field_22492 moodSound Ljava/util/Optional;
+	FIELD field_22493 additionsSound Ljava/util/Optional;
+	METHOD <init> (IIILjava/util/Optional;Ljava/util/Optional;Ljava/util/Optional;Ljava/util/Optional;)V
+		ARG 1 fogColor
+		ARG 2 waterColor
+		ARG 3 waterFogCOlor
+		ARG 4 particleConfig
+		ARG 5 loopSound
+		ARG 6 moodSound
+		ARG 7 additionsSound
 	METHOD method_24387 getFogColor ()I
 	METHOD method_24388 getWaterColor ()I
 	METHOD method_24389 getWaterFogColor ()I
 	METHOD method_24390 getParticleConfig ()Ljava/util/Optional;
+	METHOD method_24939 getLoopSound ()Ljava/util/Optional;
+		COMMENT Returns the loop sound.
+		COMMENT
+		COMMENT <p>A loop sound is played continuously as an ambient sound whenever the
+		COMMENT player is in the biome with this effect.
+	METHOD method_24940 getMoodSound ()Ljava/util/Optional;
+		COMMENT Returns the mood sound.
+		COMMENT
+		COMMENT <p>A mood sound is played once every 6000 to 17999 ticks as an ambient
+		COMMENT sound whenever the player is in the biome with this effect and near a
+		COMMENT position that has 0 sky light and less than 7 combined light.
+		COMMENT
+		COMMENT <p>Overworld biomes have the regular cave sound as their mood sound,
+		COMMENT while three nether biomes in 20w10a have their dedicated mood sounds.
```

Comment:
Needs rephrasing once we're no longer in snapshot phase

Reply:
Yeah, wip (leaving specific version ref so if the mood def changes we can come back and compare old with new)